cinspect/evaluators.py

```diff
@@ -138,8 +138,15 @@ def prepare(self, estimator, X, y, random_state=None):
         This is called by a model evaluation function in model_evaluation.
         """
         T = X[self.treatment_column]
-        assert self.treatment_val in T
-        assert self.control_val in T
+        # TODO: these are important but fragile assumptions
+        # I think it would make more sense to make an assertion about the density of points
+        # in the *neighbourhood* of the values
```

Comment:
I guess it depends on how general we want the tool to be. Continuous treatments are well-defined and useful (they're what I did my summer scholar project on), so maybe we don't want to eliminate them from the API/interface. We could state in the documentation that the current iteration of the code is designed with binary treatments in mind, and put continuous treatments as a long-term stretch goal? I can imagine a project might come up where it would be natural to use them

Reply:
We definitely support continuous treatments - but in the other evaluators right? Do we need to support them in the `BinaryTreatmentEffect` evaluator when we could just enforce inputting a bool/int array indicating treatment status? I suppose this current interface allows us to assign other values to control and treatment other than 0/1... do you think that is useful? Perhaps we can solve this in a future issue.

Reply:
That does make sense -- I forgot the context we were in. I don't see any harm in locking it down here and generalising it later if we want!